### PR TITLE
Youtube project

### PR DESCRIPTION
**Pull request by abreham1222**
Added youtube project
1. Added index.html
2. Added style.css
3. Added important image for the file
4. Added linters for github actions

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+  margin-right: 8px;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+}
+
+.nav-ico {
+  width: 32px;
+  height: 32px;
+}
```
Comment (abreham1222) [inline style.css:84]: Please add the same Icon as Youtube and margins to your design and try to stick as much as possible to the given design
youtube 1

File: style.css [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+
+.btn-thumbs {
+  position: relative;
+  font-size: 10px;
+  left: 465px;
+  bottom: 30px;
+  margin-right: 180px;
+  cursor: pointer;
+}
+
```
Comment (abreham1222) [inline style.css:137]: Please add the right widths and margins to your design and try to stick as much as possible to the youtube page
 design.
like btn

File: style.css [hunk 3]
```diff
@@ -0,0 +1,467 @@
...
+  margin-left: 64px;
+}
+
+.channel-description {
+  margin-left: 75px;
+  color: rgb(96, 96, 96);
+  font-size: 13px;
+  font-weight: 400;
+  line-height: 18px;
+}
```
Comment (abreham1222) [inline style.css:184]: Please add the right font size and right margins to "subscribe" to your design and try to stick as much as possible to the given design.
kindly Increase font size of description page
font

File: style.css [hunk 4]
```diff
@@ -0,0 +1,467 @@
...
+
+.side-bar:first-child {
+  margin-top: 0;
+  margin-bottom: 8px;
+}
+
+.side-bar a {
+  display: flex;
+  text-decoration: none;
+}
```
Comment (abreham1222) [inline style.css:373]: Please add the side video position to the left side video player and margins to your design and try to stick as much as possible. 
Kindly add image preview to your left side position of videos.
size

File: style.css [hunk 5]
```diff
@@ -0,0 +1,467 @@
...
+  bottom: 8px;
+  right: 4px;
+  color: white;
+  padding: 2px 4px;
+  border-radius: 3px;
+  background-color: black;
+  font-size: 12px;
+  font-weight: 500;
+  opacity: 0.9;
+}
```
Comment (abreham1222) [inline style.css:425]: Please add the correct(straight) video length to your design and try to stick as much as possible.
lenght

**Review by abreham1222 — COMMENTED** [review]
### Required Changes ♻️

Hi @abreham1222 ,

Good job so far! 👍
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlights
✔️ No linters error.
✔️ Github flow used properly.
✔️ Descriptive PR title & description.

**Review by omar-muhamad — APPROVED** [review]
# Approved :heavy_check_mark:
### Hi @abreham1222,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You have more than 2 inline comments.
- You asked for changes in a friendly and polite way.
- your comments are clear and easy to understand.
- Most times you explained why the reviewee needs the change.

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**